packages/longhorn/longhorn-1.3/generated-changes/overlay/crds/crds.yaml

```diff
@@ -17,157 +18,158 @@ spec:
-      name: Node
-      type: string
-    - description: The disk the backing image file will be prepared on
-      jsonPath: .spec.diskUUID
-      name: DiskUUID
-      type: string
-    - jsonPath: .metadata.creationTimestamp
-      name: Age
-      type: date
-    name: v1beta2
-    schema:
-      openAPIV3Schema:
-        description: BackingImageDataSource is where Longhorn stores backing image data source object.
-        properties:
-          apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-            type: string
-          kind:
-            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-            type: string
-          metadata:
-            type: object
-          spec:
-            description: BackingImageDataSourceSpec defines the desired state of the Longhorn backing image data source
-            properties:
-              checksum:
-                type: string
-              diskPath:
-                type: string
-              diskUUID:
-                type: string
-              fileTransferred:
-                type: boolean
-              nodeID:
-                type: string
-              parameters:
-                additionalProperties:
-                  type: string
-                type: object
-              sourceType:
-                enum:
-                - download
-                - upload
-                - export-from-volume
-                type: string
-              uuid:
-                type: string
-            type: object
-          status:
-            description: BackingImageDataSourceStatus defines the observed state of the Longhorn backing image data source
-            properties:
-              checksum:
-                type: string
-              currentState:
-                type: string
-              ip:
-                type: string
-              message:
-                type: string
-              ownerID:
-                type: string
-              progress:
-                type: integer
-              runningParameters:
-                additionalProperties:
-                  type: string
-                nullable: true
-                type: object
-              size:
-                format: int64
-                type: integer
-              storageIP:
-                type: string
-            type: object
-        type: object
-    served: true
-    storage: true
-    subresources:
-      status: {}
+    - additionalPrinterColumns:
```

Comment:
The format is different from what we usually use, so suggest using the same formatting by disable having indention for list item.

Reply:
Thank you. Fixed